.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Get all official build targets
         id: get-targets
-        run: echo "::set-output name=targets::$(make targets-ci-print | jq -R -c 'split(" ")')"
+        run: echo '{targets}={$(make targets-ci-print | jq -R -c split(" "))}' >> $GITHUB_OUTPUT
```

Comment:
Need some help here as this line is failing the build, however I use the ` and "

Reply:
Seeing the prior line, maybe you are missing the single quotes around `split(" ")`. The comand must be between quotes. But maybe for doing this you will need to change the external single quotes by double ones. Maybe something like this:

```suggestion
        run: echo "{targets}={$(make targets-ci-print | jq -R -c 'split(" ")')}" >> $GITHUB_OUTPUT
```

Reply:
Thanks. Trying this now